storage/backup/backup-cli/src/metadata/mod.rs

```diff
@@ -85,6 +206,15 @@ impl Metadata {
 }
 
 #[derive(Clone, Deserialize, Serialize, Eq, PartialEq, Ord, PartialOrd)]
+pub struct EpochEndingBackupMetaRange {
```

Comment:
This is just a data structure making the code easy to write and read, which is essentially having multiple lines of json in one Json.  This is not for merging the manifest files. But as you mentioned, it can be extended to point just one manifest file in next step.

Reply:
see my other comment :)

Reply:
addressed